src/utils/common.h

```diff
@@ -24,6 +24,19 @@ inline void checkParam<std::string>(std::string const &param, std::string const
                 (error + "Expected: " + expectedParam +", got: " + param);
     }
 }
+
+inline std::string getCanonicalRootPath(std::string const &path)
+{
+    // Get absolute path to executable
+    std::string canonicalPath = std::string(canonicalize_file_name("/proc/self/exe"));
+
+    // Remove everything in path until elrond-sdk-erdcpp directory and concatenate it with the path
+    // Use rfind because github action runs into elrond-sdk-erdcpp/elrond-sdk-erdcpp folder
```

Comment:
This works either way, `rfind` just makes sure that if multiple `elrond-sdk-erdcpp` folders exist, it will choose the utter most right one.
Agree regarding github customizing, but couldn't find how to change the root path. Do you know what's the trick?

Reply:
Building, then copying the binary to a custom location (and changing the directory) is supported by Github workflows. Even though changing the path of the repository is a bit trickier (I think you need to configure https://github.com/actions/checkout).